osgar/drivers/kloubak.py

```diff
@@ -126,6 +124,16 @@ class RobotKloubak(Node):
     def __init__(self, config, bus):
         super().__init__(config, bus)
         bus.register('pose2d', 'emergency_stop', 'encoders', 'can')
+        global WHEEL_DISTANCE
```

Comment:
I would dedicate function like `setup_global_const(config)` to reduce the size of `__init__()`

Reply:
Ok, why not. I'll put it outside the class. Ok?

Reply:
OK